### ISSUE TITLE
Stereo Approaches 

### ISSUE THREAD

**Issue by ST6-Bravo** [vatpac-technology/sops#411]
Since stereo approaches are common for most military airbases, should we have a dedicated section in each TCU, or a common piece in controller skills?

**Comment by mattkelly4** [comment]
@ST6-Bravo can you give a bit of insight into what a stereo approach is?

**Comment by ST6-Bravo** [comment]
Its how fast jets typically arrive back from an airport, i.e at Willy they track in from gate 6 of m550, hit the airport at F125 and then track out for 15nm and chuck a hook in for RWY 12.

**Comment by alphadelta332** [comment]
> Its how fast jets typically arrive back from an airport, i.e at Willy they track in from gate 6 of m550, hit the airport at F125 and then track out for 15nm and chuck a hook in for RWY 12.

Yeah sounds like this would be something that's good to add to the Military Procedures Controller Skills page. Go for it! Cheers

**Comment by ST6-Bravo** [comment]
WIll do, just wondering how you want me explain the different airports versions?

**Comment by alphadelta332** [comment]
> WIll do, just wondering how you want me explain the different airports versions?

So each airport has its own version of the stereo approach procedure?

**Comment by ST6-Bravo** [comment]
> > WIll do, just wondering how you want me explain the different airports versions?
> 
> So each airport has its own version of the stereo approach procedure?

Yes, but its a common approach though, just differing alts of entry and gates

**Comment by ST6-Bravo** [comment]
Its a bit like intial and pitches, they slighty vary depending on airport like YWLM is F125 while YPTN is F190

**Comment by alphadelta332** [comment]
> Its a bit like intial and pitches, they slighty vary depending on airport like YWLM is F125 while YPTN is F190

Okay in that case, probably best for them to go in the individual local instructions, as their own subheading

**Comment by ST6-Bravo** [comment]
Copy, Ill put it in for only ACG based airports as they are the most likely to use them

**Comment by mattkelly4** [comment]
Is there common material which could be included in the Military Procedures section, as a bit of background on what the term means and how it is implemented, and then we can include the location-specific procedures on each aerodrome page?

**Comment by ST6-Bravo** [comment]
Yeah, common details would be tracking from a gate and then aiming for an initial and pitch for the duty runway if weather permits.
